From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Mon, 14 May 2012 09:47:05 +0000
Subject: [PATCH 4/7] net/codel: Add missing #include <linux/prefetch.h>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit ce5b4b977127ee20c3f9c3fd3637cd3796f649f5 upstream.

m68k allmodconfig:

net/sched/sch_codel.c: In function ‘dequeue’:
net/sched/sch_codel.c:70: error: implicit declaration of function ‘prefetch’
make[1]: *** [net/sched/sch_codel.o] Error 1

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Acked-by: Eric Dumazet <edumazet@google.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 net/sched/sch_codel.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-3.2.46/net/sched/sch_codel.c
===================================================================
--- linux-3.2.46.orig/net/sched/sch_codel.c	2013-07-26 19:22:31.000000000 +0000
+++ linux-3.2.46/net/sched/sch_codel.c	2013-07-26 19:22:36.000000000 +0000
@@ -46,6 +46,7 @@
 #include <linux/kernel.h>
 #include <linux/errno.h>
 #include <linux/skbuff.h>
+#include <linux/prefetch.h>
 #include <net/pkt_sched.h>
 #include <net/codel.h>
 
